components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,765 @@
+
+## 3. Continuous Equations
+
+The continuous form of the conservation equations are as follows [Kundu et al. 2024](https://www.amazon.com/Fluid-Mechanics-Pijush-K-Kundu/dp/012405935X), chapter 4, eqns 4.7 and 4.22. This is before any assumptions are made, so this is a compressible, non-hydrostatic, non-Boussinesq fluid. Here all variables are a function of $(x,y,z)$, ${\bf u}_{3D}$ denotes the three-dimensional velocity vector, ${\bf u}_{3D} \otimes {\bf u}_{3D} = {\bf u}_{3D}{\bf u}_{3D}^T$ is the tensor product, $\nabla_{3D}$ is the three-dimensional gradient, $D/Dt$ is the material derivative, and other variables defined in the [Variable Definition Section](#variable-definitions) below.
+
+momentum:
+
+$$
+\frac{D \rho {\bf u}_{3D} }{D t} \equiv
+\frac{\partial \rho {\bf u}_{3D}}{\partial t}
+ + \nabla_{3D} \cdot \left( \rho {\bf u}_{3D} \otimes {\bf u}_{3D}  \right)
+  = - \nabla_{3D} p
++ \rho {\bf D}^u + \rho {\bf F}^u
+$$ (continuous-momentum)
+
+mass:
+
+$$
+\frac{D \rho}{D t} \equiv
+\frac{\partial \rho }{\partial t}
+ + \nabla_{3D} \cdot \left( \rho  {\bf u}_{3D}  \right)
+= 0
+$$ (continuous-mass)
+
+tracers:
+
+$$
+\frac{D \rho \varphi }{D t} \equiv
+\frac{\partial \rho \varphi}{\partial t}
+ + \nabla_{3D} \cdot \left( \rho \varphi {\bf u}_{3D}  \right)
+= D^\varphi + Q^\varphi
+$$ (continuous-tracer)
+
+Here we have express the following terms as a general operators, with examples of specific forms provided below: the dissipation ${\bf D}^u$, momentum forcing ${\bf F}^u$, tracer diffusion $D^\varphi$, and tracer sources and sinks $Q^\varphi$. The momentum equation can be rewritten using the product rule on $\rho {\bf u}$, mass conservation, and dividing by $\rho$, as:
+
+$$
+\frac{D {\bf u}_{3D} }{D t} \equiv
+\frac{\partial {\bf u}_{3D}}{\partial t}
++ {\bf u}_{3D}\cdot \nabla_{3D} {\bf u}_{3D}
+  = - \frac{1}{\rho} \nabla_{3D} p
++ {\bf D}^u + {\bf F}^u
+$$ (continuous-momentum-rho)
+
+Geophysical fluids such as the ocean and atmosphere are rotating and stratified, and horizontal velocities are orders of magnitude larger than vertical velocities. It is therefore convenient to separate the horizontal and vertical as ${\bf u}_{3D} = \left( {\bf u}, w \right)$ and $\nabla_{3D} = \left( \nabla_z, d/dz \right)$ where $z$ is the vertical direction in a local coordinate frame on the Earth's surface and $w$ is the vertical velocity. The $z$ subscript on $\nabla_z$ is to remind us that this is the true horizontal gradient (perpendicular to $z$), as opposed to gradients within tilted layers used in the following section. The gravitational force $g {\bf k}$ is included in ${\bf F}^u$. The rotating frame of reference results in the Coriolis force $f {\bf k} \times {\bf u} \equiv f {\bf u}^\perp$, where $f$ is the Coriolis parameter and ${\bf u}^\perp$ is the horizontal velocity rotated $90^o$ from $\bf u$. See any textbook in the [References](#references) for a full derivation.
```

Comment:
I find the `z` notation in $\nabla_z$ confusing for a horizontal gradient. Correct me if this is standard notation. Maybe add the perpendicular symbol?

Reply:
The standard notation for tilted layers is $\nabla_r$, as in [Adcroft and Hallberg (2006)](https://www.sciencedirect.com/science/article/pii/S1463500305000090) or $\nabla_{\sigma}$ in [Griffies et al (2000)](http://sciencedirect.com/science/article/pii/S1463500300000147). Since $r$ is the tilted near-vertical coordinate, I used $\nabla_z$ for the exact vertical $z$ by analogy. I think $\nabla_\perp$ is confusing because the notation is not clear as to whether it is perpendicular to $z$ or $r$.

Reply:
If the confusion is specific to z, could we include $\nabla_{z=cst}$ just to be explicit? I agree with both this is a common notation, but I also agree that it can get confusing to people unfamiliar with it. The other option is $\nabla|_z$

Reply:
Yes, after thinking about this for a couple of days, I agree that $\perp$ isn't great.  For example, we use $u^\perp$ to indicate velocity perpendicular to $u$, which is quite a different use.  While it's $\nabla_z$ wasn't immediately very intuitive to me, I do agree that it's consistent with $\nabla_r$, and that convention is established in the literature.

Reply:
@alicebarthel, I could get behind your second suggestion of $\nabla|_z$.